CMakeLists.txt

```diff
@@ -129,6 +129,10 @@ endif()
 
 include(set_compile_flags)
 
+if(ERF_AW_MULTIBLOCK)
```

Comment:
Could you change `AW` to `AMR_WIND` just so we keep the style with other ifdefs?

Reply:
Is this option coming in externally?

Reply:
yes, set by the driver build script: https://github.com/erf-model/erf-amrwind-driver/blob/main/Build/megabuild.sh